README.md

```diff
@@ -111,7 +111,20 @@ Write a conditional statement that...
 ### Your solution here:
 8.  Write your javascript solution below
 ```js
-// your answer here
+for (let i = 1; i <= 100; i++) {
+    let isDividibleByThree = i % 3 === 0;
+    let isDivisibleByFive = i % 5 === 0;
+
+    if (isDividibleByThree && isDivisibleByFive) {
+        console.log('FizzBuzz');
+    } else if (isDividibleByThree) {
+        console.log('Fizz');
+    } else if (isDivisibleByFive) {
+        console.log('Buzz');
+    } else {
+        console.log(i);
+    }
+}
 ```
 
 #### BONUS
```

Comment:
i see you removed the bonuses

Reply:
I thought the bonus is a requirement I can implement in the last part of the homework "Conditionals", so that why I did remove it.

9. By using the loop to achieve the condition runs on every number from 0 to 100 
10. I did miss it, I was plan to achieve it with the same question.

Thank you for your check-up and for your following, keep it up.